for_curators/requirements.txt

```diff
@@ -0,0 +1,7 @@
+jupyter==1.0.0
+jupyter-client==7.3.0
+jupyter-console==6.4.3
+jupyter-core==4.10.0
+jupyterlab-pygments==0.2.2
+jupyterlab-widgets==1.1.0
```

Comment:
I think you can just install the `notebook` package and then you're done. `notebook` has all these dependencies internally. (i.e. `pip install notebook`).

Reply:
noted!